src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+import core.basesyntax.figures.Figure;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[6];
+        figures[3] = new Square("black", "Square", 5);
+        figures[4] = new RightTriangle("yellow", "Right triangle", 5, 4);
```

Comment:
Ensure that the constructor parameters for `Square` and `RightTriangle` are correct and align with the expected parameters. Verify if the `name` parameter is necessary and correctly used.